ui/src/components/Liquidity/Liquidity.jsx

```diff
@@ -114,13 +104,12 @@ const Liquidity = () => {
               opacity: 1,
               boxShadow: '0px 0px 99px var(--color-secondary)',
             }}
-            transition={{ duration: 0.8 }}
-            layout
+            transition={{ duration: 0.4 }}
           >
             <div className="flex flex-col justify-between  gap-2 ">
               <h1 className="text-2xl font-semibold">Liquidity</h1>
               <h2 className="text-gray-500 ">
-                All liquidity pairs currently use {centralInfo?.petname}
+                All liquidity pairs currently use RUN.
```

Comment:
If it didn't load yet, it was rendering with a placeholder value that looked confusing. I figured this is not something that needs to be dynamic since everyone should understand the token as the same symbol.